packages/cli/lib/projects.js

```diff
@@ -807,6 +813,21 @@ const createProjectComponent = async (component, name) => {
   );
 };
 
+const showPlatformVersionWarning = async (accountId, projectConfig) => {
+  const platformVersion = projectConfig.platformVersion;
+
+  if (!platformVersion) {
+    const defaultVersion = await fetchDefaultVersion(accountId);
+    logger.log('');
+    logger.warn(
+      i18n(`${i18nKey}.showPlatformVersionWarning.noPlatformVersion`, {
+        defaultVersion,
+      })
+    );
+    logger.log('');
+  }
+};
+
```

Comment:
Wondering if there's some way this could all be refactored to return the default platform version so we can pass it in to the functions that need it when `projectConfig.platformVersion` doesn't exist. That way, we won't need to make a second request to get the default version in cli-lib

Reply:
I think we should keep the function that shows the warning and the function that returns the default version separate, just so we don't risk displaying the same message 15 times in the future. This is a straightforward API call that we're only using with three commands, so I don't think it's worth complicating the code. Going to resolve this, but happy to discuss over Slack.

Reply:
That's fair, I agree that that wouldn't actually be the best way to do it. It just doesn't feel great making two extra requests every time you run these commands. Might even be worth exploring a way to just edit the config if you don't have a version specified to the request only needs to be made once. @brandenrodgers curious what your thoughts are here.

Anyway, don't need to over complicate things for now - i'll approve